requirements.txt

```diff
@@ -25,3 +25,4 @@ python-Levenshtein==0.12.0
 dickens==1.0.1
 skills-utils==0.4.0
 s3fs==0.1.5
+cached-property
```

Comment:
I don't think you need this. dickens provides it, but for some reason we have to import it from 'descriptors':

`from descriptors import cachedproperty`